python/GangaDirac/Lib/RTHandlers/DiracRTHUtils.py

```diff
@@ -70,6 +75,9 @@ def dirac_outputfile_jdl(output_files):
         if outputSE != '':
             myLine = myLine.replace('###OUTPUT_SE###', str([outputSE]))
         else:
+            if empty_SE_check is True:
```

Comment:
Correct, although during development there was a more complex object here that made more sense to reduce to a bool